packs/vsphere/actions/vm_hw_cpu_mem_flex.yaml

```diff
@@ -0,0 +1,23 @@
+---
+  name: "vm_hw_cpu_mem_flex"
+  runner_type: "run-python"
+  description: "Adjust CPU and RAM allocation for a Virtual Machine"
+  enabled: true
+  entry_point: "vm_hw_cpu_mem_flex.py"
+  parameters:
+    vm_id:
+      type: "string"
+      description: "VM to Flex."
+      required: false
+    vm_name:
```

Comment:
I'm hesitant to restrict this just to the vm id.. mainly as that means what ever system/person is calling this must know that value otherwise i can't be used. As an interim i've added in extra logic so if multiple items are found matching name or ID it will raise an exception.